src/radical/pilot/utils/prof_utils.py

```diff
@@ -493,7 +493,7 @@ def get_session_profile(sid, src=None):
     if os.path.exists(src):
         # we have profiles locally
         profiles  = glob.glob('%s/*.prof'    % src)
-        profiles += glob.glob('%s/**/*.prof' % src)
+        profiles += glob.glob('%s/**/*.prof' % src, recursive=True)
```

Comment:
the updated command will include the one above, right? so we can just keep `profiles = glob.glob('%s/**/*.prof' % src, recursive=True)`

Reply:
You are right - I actually expected that *not* to be the case... :-P  Fixed!